homework/script.js

```diff
@@ -0,0 +1,39 @@
+// console.log("hello");
+
+// function fakePop(array){
+//     if(array.length > 0){
+//         const pop = array[array.length -1];
+//         array.length -= 1;
+//         return pop;
+//     }
+// }
+// const array = [1,2,3,4,10];
+// const lastItem = fakePop(array);
+// console.log(lastItem);
+// console.log(array);
+
+// function fakeFill(array, fill){
+//     for(let i = 0; i < array.length; i++){
+//         array[i] = fill;
+//     }
+//     return array;
+// }
+// const array = [1,2,3,4,5];
+// const fill = [15, 16, 17];
+// const fake = fakeFill(array, fill);
+// console.log(fake);
+
+
+// function fakeFill(array,start, finish, fill){
```

Comment:
this doesn't work the same as Array.prototype.fill. Let's try to implement the full version of the fill method.